### PR TITLE
Add back some missing moments

### PR DESCRIPTION
**Pull request by michaeloriordan**
Add moments and tests for the below distributions as part of https://github.com/pymc-devs/pymc/issues/5078

- pymc.distributions.continuous.Kumaraswamy
- pymc.distributions.continuous.Exponential
- pymc.distributions.continuous.Laplace
- pymc.distributions.continuous.StudentT
- pymc.distributions.continuous.Cauchy

### CODE REVIEW

File: pymc/distributions/continuous.py [hunk 1]
```diff
@@ -1399,6 +1405,12 @@ def dist(cls, lam, *args, **kwargs):
         # Aesara exponential op is parametrized in terms of mu (1/lam)
         return super().dist([at.inv(lam)], **kwargs)
 
+    def get_moment(rv, size, lam):
```
Comment (ricardoV94) [inline pymc/distributions/continuous.py]: We are converting lam to mu in the `dist` class method, so `get_moment` should expect `mu` already and you don't need to invert it (the same happens with the logcdf below). 

That's why your tests are failing for the exponential.

**Review by ricardoV94 — CHANGES_REQUESTED** [review]

File: pymc/distributions/continuous.py (hunk 1 above)
Comment (michaeloriordan) [reply pymc/distributions/continuous.py]: Ah that makes sense, thanks! I'll make that change.

**Review by ricardoV94 — APPROVED** [review]
Looks good to me. Waiting to see if the tests pass

**Comment by ricardoV94** [comment]
Thanks once again @michaeloriordan!

**Pull request merged**